src/main/java/core/basesyntax/Factorial.java

```diff
@@ -6,6 +6,10 @@
  */
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        if (number <= 1 && number >= 0) {
+            return 1;
+        } else {
```

Comment:
Do we need this else block here? Try removing it.

Reply:
removed "else"